ballerina/websocket_service.bal

```diff
@@ -91,6 +91,7 @@ isolated service class WsService {
                 if connectionId is () {
                     return self.handleIdNotPresentInPayload(caller);
                 }
+                SubscriptionHandler handler = new(connectionId);
```

Comment:
Did we remove the support for non-protocol requests? If not, let's remove that with PR. Going forward, we should only allow the requests that adhere to the graphql-ws protocol.

Reply:
Support for non-protocol requests are not yet removed. We could address this change in a new PR. Related issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/3608